public/locales/en/translation.json

```
@@ -1 +1,9 @@
-{}
+{
+  "ContactUsPage": {
```

Use the same camelcase as for variables the first letter is small and every other first letter is capital you have done it right with "firstName" and "lastName" 
 ```suggestion
  "contactUsPage": {
```